Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'http://rubygems.org'
 gemspec
-gem 'mustermann', '< 2'
+gem 'grape-entity'
+gem 'mustermann', '< 3'
```

Comment:
v2 release fixes ruby 3 issues.  could bump to v3 but it drops ruby <2.6 support...
https://github.com/sinatra/mustermann

Reply:
please use `>= 3`

Reply:
we are using >= 2.7 ourselves https://github.com/bsm/grape-apidoc/pull/22/files#diff-94c26ceeee2765addb46004091a81a08ff9edc5dccefa6b94692754973c56c97R13